sublime-package.json

```diff
@@ -11,6 +11,11 @@
               "default": false,
               "markdownDescription": "Run the server's formatProvider (if supported) on a document before saving. This option is also supported in syntax-specific settings and/or in the `\"settings\"` section of project files."
             },
+            "confirm_global_rename": {
```

Comment:
This is not in the right place for the `"confirm_global_rename"`.

Currently it is put in JSONSchema definitions,
but you want to put it here https://github.com/sublimelsp/LSP/blob/60d725264e8fafe9111cb1ab8373f9829cc63f79/sublime-package.json#L288

Reply:
This should be fixed now.